hnn_core/parallel_backends.py

```diff
@@ -0,0 +1,265 @@
+"""Parallel backends"""
+
+# Authors: Blake Caldwell <blake_caldwell@brown.edu>
+#          Mainak Jas <mainakjas@gmail.com>
+
+from warnings import warn
+
+_backend = None
```

Comment:
global variables in all caps according to pep8 as far as I remember. It's easier to know that they can change in other functions